### ISSUE TITLE
azcopy benchmark was failed and test data remains (10.13.0)

### ISSUE THREAD

**Issue by yuriwoof** [Azure/azure-storage-azcopy#1623]
##### Note: The version is visible when running AzCopy without any argument

10.13.0

### Which platform are you using? (ex: Windows, Mac, Linux)

Windows 10

##### Note: Please remove the SAS to avoid exposing your credentials. If you cannot remember the exact command, please retrieve it from the beginning of the log file.

```
azcopy benchmark 'https://labazcopysaejp.blob.core.windows.net/container(SAS)'
```

SAS created under below condition
![image](https://user-images.githubusercontent.com/1770342/142228009-688b9da8-0455-4e48-a1c7-fd665f27af41.png)

### What problem was encountered?

That operation didn't finish successfully.

```
PS C:\Users\yurio> azcopy benchmark 'https://labazcopysaejp.blob.core.windows.net/container?(SAS)'
INFO: The benchmark feature is currently in Preview status.
INFO: Benchmarking uploads to https://labazcopysaejp.blob.core.windows.net/container/benchmark-ae6be876-ec81-b24d-7025-213fecab7bd2.
INFO: Scanning...
INFO: Any empty folders will not be processed, because source and/or destination doesn't have full folder support

Job ae6be876-ec81-b24d-7025-213fecab7bd2 has started
Log file is located at: C:\Users\yurio\.azcopy\ae6be876-ec81-b24d-7025-213fecab7bd2.log

INFO: Trying 4 concurrent connections (initial starting point)
INFO: Trying 16 concurrent connections (seeking optimum)
INFO: Trying 4 concurrent connections (backing off)
INFO: Trying 8 concurrent connections (seeking optimum)
INFO: Trying 16 concurrent connections (seeking optimum)
INFO: Trying 8 concurrent connections (backing off)
INFO: Trying 9 concurrent connections (seeking optimum)
INFO: Trying 11 concurrent connections (seeking optimum)
INFO: Trying 9 concurrent connections (at optimum)
INFO:
INFO: Automatic concurrency tuning completed. Recording of performance stats will begin now.
INFO:
INFO: *** After a minute or two, you may cancel the job with CTRL-C to trigger early analysis of the stats. ***
INFO: *** You do not need to wait for whole job to finish.                                                  ***
INFO:
99.9 %, 98 Done, 0 Failed, 2 Pending, 0 Skipped, 100 Total, 2-sec Throughput (Mb/s): 41.6323


Job ae6be876-ec81-b24d-7025-213fecab7bd2 summary
Elapsed Time (Minutes): 15.4534
Number of File Transfers: 100
Number of Folder Property Transfers: 0
Total Number of Transfers: 100
Number of Transfers Completed: 100
Number of Transfers Failed: 0
Number of Transfers Skipped: 0
TotalBytesTransferred: 26214400000
Final Job Status: Completed

Diagnostic stats:
IOPS: 3
End-to-end ms per request: 2392
Network Errors: 0.00%
Server Busy: 0.00%

Performance benchmark results:
Note: The benchmark feature is currently in Preview status.

Main Result:
  Code:   NetworkIsBottleneck
  Desc:   Network bandwidth appears to be the key factor governing performance.
  Reason: No other factors were identified that are limiting performance, so the bottleneck is assumed to be available network bandwidth. The available network bandwidth is the portion of the installed bandwidth that is not already used by other traffic. Throughput of 230 Mega bits/sec was obtained with 9 concurrent connections.

This benchmark tries to find optimal performance, computed without touching any local disk. When reading
and writing real data, performance may be different. If disk limits throughput, AzCopy will display a message on screen.

INFO: Any empty folders will not be processed, because source and/or destination doesn't have full folder support
99.9 %, 98 Done, 0 Failed, 2 Pending, 0 Skipped, 100 Total, 2-sec Throughput (Mb/s): 41.6323panic: couldn't create job part plan file "ae6be876-ec81-b24d-7025-213fecab7bd2--00000.steV16": open C:\Users\yurio\.azcopy/plans/ae6be876-ec81-b24d-7025-213fecab7bd2--00000.steV16: The requested operation cannot be performed on a file with a user-mapped section open.

goroutine 240 [running]:
github.com/Azure/azure-storage-azcopy/v10/ste.JobPartPlanFileName.Create(0xc0006bed00, 0x32, 0xae6be87600000000, 0x3f212570b24dec81, 0xd27babec, 0x300000000000001, 0x1, 0x0, 0x0, 0x0, ...)
        /home/vsts/work/1/s/ste/JobPartPlanFileName.go:134 +0x2905
github.com/Azure/azure-storage-azcopy/v10/ste.ExecuteNewCopyJobPartOrder(0xae6be87600000000, 0x3f212570b24dec81, 0xd27babec, 0x300000000000001, 0x1, 0x0, 0x0, 0x0, 0xc0003da150, 0x65, ...)
        /home/vsts/work/1/s/ste/init.go:165 +0xb9
github.com/Azure/azure-storage-azcopy/v10/cmd.inprocSend(0x1b7d465, 0x10, 0x19b91a0, 0xc00016a000, 0x19b91e0, 0xc042749998, 0x1, 0xc042749998)
        /home/vsts/work/1/s/cmd/rpc.go:44 +0x938
github.com/Azure/azure-storage-azcopy/v10/cmd.glob..func6(0x1b7d465, 0x10, 0x19b91a0, 0xc00016a000, 0x19b91e0, 0xc042749998)
        /home/vsts/work/1/s/cmd/rpc.go:36 +0x68
github.com/Azure/azure-storage-azcopy/v10/cmd.(*copyTransferProcessor).sendPartToSte(0xc0149b6360, 0x0, 0x0, 0x0)
        /home/vsts/work/1/s/cmd/zc_processor.go:191 +0x8d
github.com/Azure/azure-storage-azcopy/v10/cmd.(*copyTransferProcessor).dispatchFinalPart(0xc0149b6360, 0x1b8236d, 0x14, 0x0)
        /home/vsts/work/1/s/cmd/zc_processor.go:167 +0x45
github.com/Azure/azure-storage-azcopy/v10/cmd.newRemoveEnumerator.func2(0xc096b6c4e0, 0x0)
        /home/vsts/work/1/s/cmd/removeEnumerator.go:83 +0x3e
github.com/Azure/azure-storage-azcopy/v10/cmd.(*CopyEnumerator).enumerate(0xc042816600, 0xc042816600, 0x0)
        /home/vsts/work/1/s/cmd/zc_enumerator.go:682 +0xa4
github.com/Azure/azure-storage-azcopy/v10/cmd.(*CookedCopyCmdArgs).processCopyJobPartOrders(0xc000080c00, 0x0, 0x0)
        /home/vsts/work/1/s/cmd/copy.go:1410 +0x919
github.com/Azure/azure-storage-azcopy/v10/cmd.(*CookedCopyCmdArgs).process(0xc000080c00, 0xc000030000, 0x1cf7800)
        /home/vsts/work/1/s/cmd/copy.go:1157 +0xc5
github.com/Azure/azure-storage-azcopy/v10/cmd.(*CookedCopyCmdArgs).launchFollowup.func1(0xc0989522a4, 0xc0006b0400)
        /home/vsts/work/1/s/cmd/copy.go:1499 +0x79
created by github.com/Azure/azure-storage-azcopy/v10/cmd.(*CookedCopyCmdArgs).launchFollowup
        /home/vsts/work/1/s/cmd/copy.go:1496 +0x66
```

### How can we reproduce the problem in the simplest way?

Install latest Azcopy (10.13.0) and execute "azcopy benchmark"

### Have you found a mitigation/solution?

Not yet.

**Comment by mohsha-msft** [comment]
Fixed in AzCopy Release `10.14.0`

**Comment by mohsha-msft** [comment]
Fixed in AzCopy Release `10.14.0`

**Comment by mohsha-msft** [comment]
Fixed in AzCopy Release `10.14.0`
